test/geocode.js

```diff
@@ -59,6 +59,26 @@ test('MapboxClient#geocodeForward', function(t) {
     });
   });
 
+  t.test('options.proximity rounding', function(t) {
+    var client = new MapboxClient(process.env.MapboxAccessToken);
+
+    var tester = { client: function(opts) {
+      opts.params.proximity.split(',').forEach(function(coord, i) {
+        t.ok((coord.split('.')[1].length <= 3), 'proximity coordinate [' + i + '] precision <= 3');
+      });
+      t.ok(opts.params.proximity, 'proximity is set')
+      opts.callback();
+    }};
+
+    t.ok(client);
+    client.geocodeForward.apply(tester, ['Paris', {
+      proximity: { latitude: 33.6875431, longitude: -95.4431142 }
```

Comment:
it looks like this should specify a `precision` option but it doesn't? the opts.precision handling is not covered by tests in this branch.